src/rust/ide/view/graph-editor/src/component/node/output/area.rs

```diff
@@ -130,12 +130,12 @@ ensogl::define_endpoints! {
     }
 
     Output {
-        on_port_press        (Crumbs),
-        on_port_hover        (Switch<Crumbs>),
-        on_port_type_change  (Crumbs,Option<Type>),
-        port_size_multiplier (f32),
-        body_hover           (bool),
-        tooltip              (tooltip::Style),
+        on_port_press         (Crumbs),
+        on_port_hover         (Switch<Crumbs>),
+        on_port_type_change   (Crumbs,Option<Type>),
+        port_size_multiplier  (f32),
+        body_hover            (bool),
+        port_label_visibility (bool),
```

Comment:
What port labels have to do with types under the node? This change seems completely unrelated to the subject of this PR.